src/signature.c

```diff
@@ -444,12 +446,12 @@ static int validate_certificate(X509 *cert, const char *certificate_path, const
 
 	/* create a verification context and initialize it */
 	if (!(verify_ctx = X509_STORE_CTX_new())) {
-		error("Failed X509_STORE_CTX_new() for %s\n", certificate_path);
+		debug("Failed X509_STORE_CTX_new() for %s\n", certificate_path);
```

Comment:
why are we changing these from error to debug

Reply:
I explained that in the commit message:

"When performing signature verification to the MoM or to the version
file, if there is any error during the process, a series of errors are
printed to stderr. Many of these error messages are duplicated and most
of them are too detailed.

This commit filters those detailed messages so they are only seen if the
user is using the --debug flag, showing only a few more general
error messages about the signature failure."